src/EnergyPlus/InputProcessing/InputProcessor.cc

```diff
@@ -898,6 +898,40 @@ void InputProcessor::setObjectItemValue(EnergyPlusData &state,
     }
 }
 
+const json &InputProcessor::getObjectItem(EnergyPlusData &state, std::string_view ObjType, std::string_view ObjName)
+{
+    auto objectInfo = ObjectInfo();
```

Comment:
This is a confusing way to create a local object. `ObjectInfo objectInfo(ObjType, ObjName);` would be clearer, shorter, and probably faster. Also, do you even need this object for anything other than looking for "unused inputs"? And how critical is it to track unused inputs? It seems like something that could add not-insignificant overhead if used everywhere.

Reply:
I'm copying getObjectItem. 

Also:

* the ctor of ObjectInfo expects a std::string, not a std::string_view,  is `auto objectInfo = ObjectInfo(std::string{ObjType}, std::string{ObjName});` really better?
* Both arguments are of the same type (std::string), so it's kinda easy to mess up the order and not realize you messed up

Unused inputs are reported when DisplayUnusedObjects is enabled in Output:Diagnostics. If not enabled, you still get a warning saying you have unused objects: https://github.com/NREL/EnergyPlus/blob/d7f7f7689e0e2804495811c0c20567f59b5d82f8/src/EnergyPlus/InputProcessing/InputProcessor.cc#L1771-L1775

Reply:
Done in 13c4b06f7b8a4f79c71fa9947d686ae093553be0